### DISCUSSION TITLE
remove duplicates in English

### DISCUSSION THREAD

**Discussion by alexeysvyridov** [WeblateOrg/weblate#5632] [Q&A]
In english translation i have many  duplicates example:

    "Early alert* when resident leaves bed during sleep periods FROM1 - TO1 and FROM2 - TO2": "",
    "Early alert* when resident leaves bed during sleep periods FROM1 - TO1 and FROM2 - TO2.": "Frühwarnung * wenn resident Blätter während Schlafperioden Bett FROM1 - TO1 und FROM2 - TO2.",
    "Early alert* when resident leaves bed during sleep periods FROM1-TO1 and FROM2-TO2.": "",

and i can't remove their from language component because it "read only"
![image](https://user-images.githubusercontent.com/33163048/110758990-cd1d1b80-8255-11eb-9374-cedd542b9ddb.png)
 
How can I clean them ?

**Comment by nijel** [comment]
Depending on how your translations are configured:

* For monolingual, just remove the source string, either from the file directly or using Weblate (if you have enabled editing of the base file)
* For bilingual, the source language is just a compilation of strings present in the translations. Remove the unwanted strings from the translations.
